core/conn/security_dll/LICENSE

```diff
@@ -1,127 +0,0 @@
-
```

Comment:
Overall the changes look very good, my only question is about this deleted file. Could you explain why we were able to delete this?